deploy/role.yaml

```diff
@@ -32,49 +34,45 @@ rules:
   - list
   - watch
   - patch
-  - update
 - apiGroups:
   - extensions
   - apps
   resources:
   - deployments
   - replicasets
   - daemonsets
-  - statefulsets
   verbs:
-  - '*'
+  - create
+  - get
+  - list
+  - patch
+  - watch
 - apiGroups:
   - ""
   resources:
   - serviceaccounts
   - configmaps
-  - nodes
-  - endpoints
-  - events
-  - secrets
+  - services
   verbs:
   - create
   - get
   - patch
-  - update
   - list
   - watch
 - apiGroups:
   - ""
   resources:
-  - pods
+  - nodes
   verbs:
-  - '*'
+  - get
+  - patch
+  - update
 - apiGroups:
   - ""
   resources:
-  - services
+  - pods
   verbs:
-  - create
```

Comment:
interesting, so dont' we need to create/list/watch pods?

Reply:
SSP operator doesn't complain, that it doesn't have these permissions.

Reply:
fair enough